mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -74,6 +75,11 @@ private InputFormatConfig() {
   public static final String SNAPSHOT_TABLE = "iceberg.snapshots.table";
   public static final String SNAPSHOT_TABLE_SUFFIX = "__snapshots";
 
+  public static final String CATALOG_CONFIG_PREFIX = "iceberg.catalog.";
+  public static final String CATALOG_TYPE_TEMPLATE = CATALOG_CONFIG_PREFIX + "%s.type";
```

Comment:
I generally recommend avoiding dependencies between constants. This pattern makes it harder to search through code to find the relevant section that you want. And I think it is easier to read to just use `"iceberg.catalog.%s.type"`.

That said, I would also recommend changing this so that the code extracts catalog properties into a map (`Catalogs.getCatalogProperties`) and then get individual keys from that map using the constants in `CatalogProperties`.